packages/orbit-components/src/Accordion/index.d.ts

```diff
@@ -8,11 +8,31 @@ import * as Common from "../common/common";
 
 declare module "@kiwicom/orbit-components/lib/Accordion";
 
+/*
+  DOCS:
+  To implement Accordion component into your project you'll need to the import the Accordion and the [AccordionSection](#Accordionsection):
+  ```jsx
+  import Accordion, { AccordionSection } from "@kiwicom/orbit-components/lib/Accordion";
+  ```
+
+  After adding import into your project you can use it simply like:
+
+  ```jsx
+  <Accordion>
+    <AccordionSection>Hello World!</AccordionSection>
+  </Accordion>
+  ```
+*/
```

Comment:
nope, `DOCS:` now is always on the top of it. Because I planned that it should contain all necessary info, below only tables with props. If README.md contains several components, it will be in that order: `DOCS -> TABLE -> CHILDREN DOCS -> CHILDREN TABLE`. You suggestion is good and I will think about it, how to improve that :)